packages/platform-redis/src/redis-interceptor.service.ts

```diff
@@ -0,0 +1,36 @@
+import { ExecutionContext, Injectable } from '@nestjs/common';
+import { PATTERN_METADATA } from '@nestjs/microservices/constants';
+import { AbstractInterceptorService } from '@ogma/nestjs-module';
+
+@Injectable()
+export class RedisParser extends AbstractInterceptorService {
+  getCallPoint(context: ExecutionContext): any {
+    return this.reflector.get(PATTERN_METADATA, context.getHandler()).rpc;
```

Comment:
What's the reason for adding `.rpc`? [I think I've handled this differently](https://github.com/jmcdo29/ogma/blob/master/packages/platform-tcp/src/tcp-interceptor.service.ts#L8) in the TCP parser

Reply:
Good catch! 😅